cfme/tests/candu/test_host_graph.py

```diff
@@ -157,3 +160,81 @@ def refresh():
     view.table.clear_cache()
     table_data = view.table.read()
     compare_data(table_data=table_data, graph_data=graph_data, legends=legends)
+
+
+@pytest.mark.uncollectif(lambda provider, interval:
+                         provider.one_of(RHEVMProvider) and
+                         interval == "Daily")
+@pytest.mark.parametrize('gp_by', GROUP_BY, ids=['vm_tag'])
+@pytest.mark.parametrize('interval', INTERVAL)
+@pytest.mark.parametrize('graph_type', HOST_GRAPHS)
+def test_tagwise(provider, interval, graph_type, gp_by, candu_tag_vm, enable_candu):
+    """Test Host graphs group by VM tag for hourly and Daily
+
+    prerequisites:
+        * C&U enabled appliance
+        * C&U data collection enabled for Tag category
+        * VM should be taged with proper tag category
+
+    Steps:
+        * If possible capture historical data for host and vm
```

Comment:
Isn't this step a prerequisite?

Reply:
We can not take as `prerequisite`. prerequisite means necessary. It's optional to reduce runtime we are collecting historical data. Now, a runtime for those 45 test nearly 2.30 hr. If we remove it. we can not go for daily and hourly also it will take 30 min * provider+ other.

Reply:
Thats why I used `If possible`.

Reply:
Just remove the 'If possible' from the sentence since historical data for host and vm needs to be captured.